app/settings/site/editor.directive.js

```diff
@@ -125,6 +125,8 @@ function (
                         let userLanguage = Session.getSessionDataEntry('language');
                         if ((userLanguage === undefined || userLanguage === null) && $scope.SystemLanguage !== newSystemLanguage) {
```

Comment:
@rowasc I find this very confusing, if someone has selected a language in the language-switcher from map-view, they wont see any changes made in the settings-view if they change to another language. Could we remove the check entirely and each time a user switch language here, it actually change the language they see in the view? I know its not covered by this issue, but it confused me a lot while testing 🤔